presenters/keyword.go

```diff
@@ -11,3 +14,39 @@ type KeywordPresenter struct {
 func (kp *KeywordPresenter) FormattedCreatedAt() string {
 	return kp.Keyword.CreatedAt.Format("January 2, 2006")
 }
+
+type KeywordResult struct {
```

Comment:
With some hindsight having this struct defined below the other method seems odd 🤔 Also `KeywordResult` seems to duplicated entirely the Keyword struct which is not good for maintainability in the sense that adding an attribute in the model struct would require to modify the presenter struct here as well. 

Note that all attributes just as ID and Status are not altered to the view can still access those as: `kp.Keyword.ID` or `kp.Keyword.Status`. So there is no need to do `kp.KeywordResult.ID` 🙃 Only `NonAdwordLinks` and `topPositionAdwordLinks` are actually formatted here.

Reply:
Agree with you. Fixed in 803f92c3c436caec1a94f017310272c8e74d8820